neptune/new/exceptions.py

```diff
@@ -191,7 +238,26 @@ def __init__(
         )
 
 
-class ProjectNotFound(ExceptionWithProjectsWorkspacesListing):
+class ContainerUUIDNotFound(NeptuneException):
+    container_id: str
+    container_type: ContainerType
+
+    def __init__(self, container_id: str, container_type: ContainerType):
+        self.container_id = container_id
+        self.container_type = container_type
+        super().__init__(
+            "{} with ID {} not found. Could be deleted.".format(
+                container_type.value.capitalize(), container_id
+            )
+        )
+
+
+RunUUIDNotFound = ContainerUUIDNotFound
```

Comment:
1. We don't raise it anywhere anymore, right?
2. Could you add a comment that it's only for backward compat etc?

Reply:
I've added a comment.